src/solver/trunk.jl

```diff
@@ -14,6 +14,7 @@
 export trunk
 
 function trunk(nlp :: AbstractNLPModel;
+               logger :: AbstractLogger=NullLogger(),
```

Comment:
`trunk` and `tron` also need a `subsolver_logger` to pass to `cg`.

Reply:
This should be easy to change after Krylov update, but will break if I do it now.

Reply:
Right, ok.